src/AppBundle/Command/ReferentDumpDatabaseCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Referent\ReferentDatabaseDumper;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ReferentDumpDatabaseCommand extends ContainerAwareCommand
+{
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var ReferentDatabaseDumper
+     */
+    private $dumper;
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:referent:dump-db')
+            ->setDescription('Dumps the list of managed users, newsletters, etc. files')
+            ->addArgument('referent', InputArgument::REQUIRED, 'The adherent UUID or email address')
+            ->addArgument('type', InputArgument::OPTIONAL, 'The type of files to generate (ie. adherents)', 'all')
+        ;
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+
+        $this->io = new SymfonyStyle($input, $output);
+        $this->dumper = $this->getContainer()->get('app.referent.database_dumper');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $referent = $input->getArgument('referent');
+        $type = $input->getArgument('type');
+
+        if (!$this->io->isQuiet()) {
+            $this->io->writeln(sprintf('Starting database export for referent <comment>%s</comment> (type: <comment>%s</comment>).', $referent, $type));
+        }
+
+        $this->dumper->dump($referent, $type);
```

Comment:
If the type is not specified, it would be more useful if all the types were dumped (not only the `all` type)